common/app/views/mainEmail.scala.html

```diff
@@ -34,6 +34,11 @@
         @if(page.metadata.isFront) {
             @fragments.email.stylesheets.front(page)
             @fragments.email.stylesheets.tones()
+
+            @* This is a temporary override to test two alternative email designs 29/11/2016 *@
+            @if(request.getQueryString("format").contains("email-connected")) {
```

Comment:
No, `isEmail` is for both "email" and "email-connected" styles - I could add an `isEmailConnected` into `Requests.scala` but it didn't quite seem worth it for this